frontend/src/components/atoms/Button/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { StyledButton } from './Button.styled';
+import { IButton } from './Button.model';
+
+const Button = ({ children, onClick, ...props }: IButton) => {
+  return (
+    <StyledButton onClick={onClick} {...props}>
+      <span> {children} </span>
+    </StyledButton>
+  );
+};
```

Comment:
```suggestion
const Button = ({ children, ...props }: IButton) => {
  return (
    <StyledButton {...props}>
      <span> {children} </span>
    </StyledButton>
  );
};
```

Check if this will work as previous version